extensions/amazon-lambda-http/runtime/src/main/java/io/quarkus/amazon/lambda/http/LambdaHttpHandler.java

```diff
@@ -234,4 +238,46 @@ private boolean isBinary(String contentType) {
         return false;
     }
 
+    private Optional<Principal> getPrincipal(APIGatewayV2HTTPEvent request) {
```

Comment:
I don't agree with this approach.  This environment variable you are adding is only for testing purposes with things like SAM LOCAL, correct?  Quarkus users use our lambda integration with *BOTH* JVM and Native.    You should just check to see if the environment variable is set.  If it is, use it, if not try to extract the JWT.